### PR TITLE
1547: Add project config for koblenz

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

To be able to start the druckerei with koblenz config we need a basic project configuration

### Proposed changes

- added basic project config for koblenz
- added favicon icon
- added runConfigs for the project roles (not sure if we need anything except project admin, since cards will be created via self service portal)
- remove ability for card creation in the administration, since they will be created in the user portal

### Side effects

- none

### Resolved issues

Fixes: #1547 

### Testing
- Create users and login

### CODE REVIEW

File: .idea/runConfigurations/Create_project_admin__env_local_buildConfig_koblenz_.xml
Comment (seluianova) [inline .idea/runConfigurations/Create_project_admin__env_local_buildConfig_koblenz_.xml:1]: just wondering, for nurnberg we only have a region admin.
won't it be the same for koblenz?

File: administration/src/project-configs/bayern/config.ts [hunk 1]
```diff
@@ -83,6 +83,7 @@ const config: ProjectConfig = {
       primaryColorLight: '#c6c0d8',
     },
   },
+  cardCreation: true,
```
Comment (seluianova) [inline administration/src/project-configs/bayern/config.ts:86]: nit-picky one:
maybe to follow the style we already have
```suggestion
cardCreation: {
  enabled: false,
},
```

Comment (f1sh1918) [reply administration/src/project-configs/bayern/config.ts:86]: i think we just need this object style if we add some additional parameters/configs if it is enabled. Which we don't do here, so i think thats not needed here

**Review by steffenkleinle — APPROVED** [review]
LGTM.
Tested creating a user via CLI and UI for koblenz in firefox. Login works as expected.
I just had to switch between the projects, otherwise I get `Server nicht erreichbar`.

File: administration/src/project-configs/koblenz/pdf.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+const renderCardHash = ({ cardInfoHash }: InfoParams) => {
+  return cardInfoHash
+}
+
+// TODO 1422 Create PDF for koblenz
+
+const pdfConfiguration: PdfConfig = {
+  title: 'Koblenz-Pässe',
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply administration/src/project-configs/koblenz/pdf.ts:26]: Indeed we have a static and a dynamic card for koblenz and this is used as a doc name of the pdf. we have the same for nuernberg.

**Comment by f1sh1918** [comment]
> LGTM. Tested creating a user via CLI and UI for koblenz in firefox. Login works as expected. I just had to switch between the projects, otherwise I get `Server nicht erreichbar`.

Dunno why the switch didn't work directly. I checked that again by removing the local storage variable and start administration again. for me it works directly if i click on switch to "Koblenz"

**Pull request merged**